### DISCUSSION TITLE
File size exploded in size

### DISCUSSION THREAD

**Discussion by thany** [NickeManarin/ScreenToGif#1188] [Q&A] [answered]
I have STG 2.34.1 portable, which is about 5MB.

This new version 2.38 is 175MB, and even the "light" version is 21MB. What gives? What could be so important that has to be included, that isn't included in 2.34, to weigh in so heavy?

Also the requirement for .NET 6 is not great. Version 5 is the standard installed on most pc's. But even with such a bulky framework around, it still needs over 4 times as big an executable? (no, I wouldn't even consider the non-light version, that one is just mental)

Please don't close this again. I just want an answer. The fact this is free software, don't mean you can just do whatever, and shut your users up. I'm making this discussion becuse I care, not because I don't care!

**Comment by NickeManarin** [comment, marked as answer]
> The fact this is free software, don't mean you can just do whatever, and shut your users up.

I can, specially when the user is not being respectful and feeling entitled to an answer.
I work on my free time, and I have been doing for almost 10 years.
Receiving a message like that demanding an explanation why a software grew in size after a span of 2 years (!) is just insulting.

And you are wrong about which .NET I was using before (it was .NET Framework 4.8, an obsolete and slower version of .NET).
